qiskit_addon_sqd/qubit.py

```diff
@@ -190,40 +190,52 @@ def matrix_elements_from_pauli(
         pauli: A Pauli operator.
 
     Returns:
-        A 1D array corresponding to the nonzero matrix elements
-        A 1D array corresponding to the row indices of the elements
-        A 1D array corresponding to the column indices of the elements
-
+        - An array of ``amplitudes`` corresponding to the nonzero bitstring matrix elements
+        - An array of ``input_configuration_ids`` specifying which configurations
+          (bitstring matrix rows) are connected elements of the Pauli operator.
+        - An array of ``connected element ids`` mapping the amplitudes to their respective
```

Comment:
Unclear why we can't just sort the amplitudes before returning from this function. Leaving as-is until I see how this data structure (formerly "column indices") is used throughout rest of this module.